docs/adrs/2023-11-02-min-runners-strategy.md

```diff
@@ -0,0 +1,23 @@
+# Introducing min runner strategies
+
+**Status**: Proposed
+
+## Context
+
+Current implementation treats the `minRunners` field as the number of runners that should be running on your cluster. They can be busy running the job, or they can be idle. This ensures faster cold startup time when workflows are acquired as well as trying to use the minimum amount of runners needed to fulfill the scaling requirement.
```

Comment:
i think runner pod kind of having 3 stage:
- Running jobs
- Idle, runner online to the service
- Runner pod starting, runner is trying to be online to the service.

Reply:
Good point! I'll add it (I merged this case with the idle one as it is not busy, but I should have been more precise)